styleguide/backstop.json

```diff
@@ -445,6 +445,15 @@
       ],
       "selectorExpansion": true
     },
+    {
+      "label": "partner promo",
+      "url": "http://localhost:3000/patterns/02-molecules-partner-promo/02-molecules-partner-promo.html",
+      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/patterns/02-molecules-partner-promo/02-molecules-partner-promo.html",
+      "selectors": [
+        ".ama__partner-promo"
+      ],
+      "selectorExpansion": true
```

Comment:
You probably don't need this. The selectorExpansion is only need if more than one of the same class appears on the page and you need backstop to iterate over them each